amee/Pose pose
uint32[] edges
uint32 nodeID
uint32 type